spice/tests/vm_listen.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+
+"""Verify listening SPICE sockets of guest VM.
+"""
+import logging
+import signal
+from spice.lib import act
+from spice.lib import stest
+
+from autotest.client.shared import error
+
+from virttest import env_process
+from virttest import virt_vm
+
+from avocado.utils import process
+
+logger = logging.getLogger(__name__)
+
+
+@error.context_aware
+def run(vt_test, test_params, env):
+    """Tests for SPICE listening sockets.
+
+    Parameters
+    ----------
+    vt_test : avocado.core.plugins.vt.VirtTest
+        QEMU test object.
+    test_params : virttest.utils_params.Params
+        Dictionary with the test parameters.
+    env : virttest.utils_env.Env
+        Dictionary with test environment.
+    """
+    if test_params['start_vm'] == "no":
+        test_params['start_vm'] = "yes"
```

Comment:
Well I must change config in the runtime. How to do this? Check your email if you want to know more.